docs/developer-guide/test-e2e.md

```diff
@@ -1,21 +1,21 @@
 # E2E Tests
 
-The directory contains E2E tests and test applications. The test assume that Argo CD services are installed into `argocd-e2e` namespace or cluster in current context. One throw-away
-namespace `argocd-e2e***` is created prior to tests execute. The throw-away namespace is used as a target namespace for test applications.
+This [directory](../../test) contains E2E tests and test applications. The tests assume that Argo CD services are installed into `argocd-e2e` namespace or cluster in current context. A throw-away
+namespace `argocd-e2e***` is created prior to the execution of the tests. The throw-away namespace is used as a target namespace for test applications.
 
-The `test/e2e/testdata` directory contains various Argo CD applications. Before test execution directory is copies into `/tmp/argocd-e2e***` temp directory and used in tests as a
+The [/test/e2e/testdata](../../test/e2e/testdata) directory contains various Argo CD applications. Before test execution directory is copies into `/tmp/argocd-e2e***` temp directory and used in tests as a
 Git repository via file url: `file:///tmp/argocd-e2e***`.
 
 ## Running Tests Locally
 
-1. Start the e2e version `make start-e2e` 
-1. Run the tests: `make test-e2e`
+1. Start the e2e version `make start-e2e`
+2. Run the tests: `make test-e2e`
```

Comment:
Markdown will automatically increment when you do repeated 1s.
https://argo-cd.readthedocs.io/en/stable/developer-guide/test-e2e/#running-tests-locally